agent/lib/artemis.js

```diff
@@ -499,7 +499,7 @@ var address_settings_fields = {
     'addressFullMessagePolicy': 'FAIL',
     'slowConsumerThreshold':-1,
     'slowConsumerCheckPeriod':-1,
-    'slowConsumerPolicy':'DROP',
+    'slowConsumerPolicy':'KILL',
```

Comment:
Why this change?

Reply:
DROP has been removed.  The available options are kill & notify
https://github.com/apache/activemq-artemis/blob/master/artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/SlowConsumerPolicy.java